test/test_parse_es.py

```diff
@@ -84,6 +86,134 @@ def test_extract_number_es(self):
+            "11 may", lang='es', anchorDate=datetime(1998, 5, 1))[0],
+            datetime(1998, 5, 11))
+        # there is an issue with the months of june through september (below)
+        # hay un problema con las meses junio hasta septiembre (lea abajo)
+        self.assertEqual(extract_datetime(
+            "11 oct", lang='es', anchorDate=datetime(1998, 10, 1))[0],
+            datetime(1998, 10, 11))
+        self.assertEqual(extract_datetime(
+            "11 nov", lang='es', anchorDate=datetime(1998, 11, 1))[0],
+            datetime(1998, 11, 11))
+        self.assertEqual(extract_datetime(
+            "11 dic", lang='es', anchorDate=datetime(1998, 12, 1))[0],
+            datetime(1998, 12, 11))
+
+        self.assertEqual(extract_datetime("", lang='es'), None)
+
+    # TODO fix bug causing these tests to fail (MycroftAI/mycroft-core#2348)
+    #         reparar error de traducción preveniendo las funciones abajo de
+    #         retornar correctamente
+    #         (escribido con disculpas por un Inglés hablante)
+    #      further broken tests are below their respective working tests.
+    @unittest.skip("currently processing these months incorrectly")
+    def test_bugged_output_wastebasket(self):
+        self.assertEqual(extract_datetime(
+            "11 jun", lang='es', anchorDate=datetime(1998, 6, 1))[0],
+            datetime(1998, 6, 11))
+        self.assertEqual(extract_datetime(
+            "11 junio", lang='es', anchorDate=datetime(1998, 6, 1))[0],
+            datetime(1998, 6, 11))
+        self.assertEqual(extract_datetime(
+            "11 jul", lang='es', anchorDate=datetime(1998, 7, 1))[0],
+            datetime(1998, 7, 11))
+        self.assertEqual(extract_datetime(
+            "11 ago", lang='es', anchorDate=datetime(1998, 8, 1))[0],
+            datetime(1998, 8, 11))
+        self.assertEqual(extract_datetime(
+            "11 sep", lang='es', anchorDate=datetime(1998, 9, 1))[0],
+            datetime(1998, 9, 11))
+
+        # It's also failing on years
+        self.assertEqual(extract_datetime(
+            "11 ago 1998", lang='es')[0], datetime(1998, 8, 11))
+
+    def test_extract_datetime_relative(self):
+        self.assertEqual(extract_datetime("esta noche", anchorDate=datetime(
+            1998, 1, 1), lang='es'),
+            [datetime(1998, 1, 1, 21, 0, 0), 'esta'])
+        self.assertEqual(extract_datetime(
+            "ayer noche", anchorDate=datetime(1998, 1, 1),
+            lang='es')[0], datetime(1997, 12, 31, 21))
+        self.assertEqual(extract_datetime(
+            "el noche anteayer", anchorDate=datetime(1998, 1, 1),
+            lang='es')[0], datetime(1997, 12, 30, 21))
+        self.assertEqual(extract_datetime(
+            "el noche ante ante ayer", anchorDate=datetime(1998, 1, 1),
+            lang='es')[0], datetime(1997, 12, 29, 21))
+        self.assertEqual(extract_datetime(
+            "mañana en la mañana", anchorDate=datetime(1998, 1, 1),
```

Comment:
In proper spanish or es_ES, should be said "mañana **por** la mañana". _Some_ latam say "mañana **en** la mañana" but those usually are es_US or es_MX, more influenced by english language.